src/20230904_AaveV3_Eth_GHOBorrowRateUpdate/AaveV3_Ethereum_GHOBorrowRateUpdate_20230904.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3EthereumAssets} from 'lib/aave-address-book/src/AaveV3Ethereum.sol';
```

Comment:
I would import it this way:
`import {AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';`

Reply:
done